api/app/services/ranker.py

```diff
@@ -3,33 +3,27 @@
 import torch
 from pygaggle.model import CachedT5ModelLoader, T5BatchTokenizer
```

Comment:
remove `CachedT5ModelLoader` since we don't need it anymore?

Reply:
Done 👍